docs/dbg.md

```diff
@@ -0,0 +1,162 @@
+The commands are the interface for talking to the debugger server.
+
+```js
+const source = dbg.selectors.getSelectedSource();
+dbg.client
+  .setBreakpoint({line: 24, sourceId: source.get("id")})
+  .then(console.log)
+```
+
+### connection
+
+`dbg.connection` has several useful properties for seeing the current connection to the tab.
+If you're looking for the current threadClient, debuggerClient, or tabTarget it is your friend!
+
+```js
+dbg.connection.tab
+/*
+{
+  "title": "Backbone.js • TodoMVC",
+  "url": "http://firefox-dev.tools/debugger-examples/examples/todomvc/",
+  "id": "server1.conn12.child1/tab1",
+  "tab": {
+    "actor": "server1.conn12.child1/tab1",
+    "title": "Backbone.js • TodoMVC",
+    "url": "http://firefox-dev.tools/debugger-examples/examples/todomvc/",
+    "outerWindowID": 2147483649,
+    "consoleActor": "server1.conn12.child1/consoleActor2",
+    "inspectorActor": "server1.conn12.child1/inspectorActor3",
+    "callWatcherActor": "server1.conn12.child1/callWatcherActor4",
+  },
+  "clientType": "firefox"
+}
+*/
+
+dbg.connection.connTarget // { type: "firefox", param: "child1/tab1" }
+
+dbg.connection.tabConnection // debuggerClient, threadClient, tabTarget
+```
+
+### bpClients
+
+`dbg.bpClients` stores the current set of breakpoint clients.
+
+### services
+
+`dbg.services` has a reference to the SourceMap worker, which is available to the redux actions.
+
+`dbg.services.sourceMaps`
+
+> originalToGeneratedId, generatedToOriginalId, isGeneratedId, isOriginalId, hasMappedSource, getOriginalURLs, getGeneratedLocation, getOriginalLocation, getLocationScopes, getOriginalSourceText, applySourceMap, clearSourceMaps, startSourceMapWorker, stopSourceMapWorker
+
+### prefs
+
+`dbg.prefs` references the PreferencesHelper. You can use `dbg.prefs` to see or change the state of any pref.
+
+```js
+dbg.prefs.pauseOnExceptions // false
+dbg.prefs.pauseOnExceptions = true
+dbg.prefs.pauseOnExceptions // true
+```
+
+### features
+
+`dbg.features` references the debugger's feature flags. You can use `dbg.features` to see or change the state of any flag.
+
+```js
+dbg.features.codeCoverage // false
+dbg.features.codeCoverage = true
+dbg.features.codeCoverage // true
+```
+
+
+### helpers
+
+#### findSource
+
+Sometimes you want to quickly grab a source object from the store, but you dont know the full url or source id. The `findSource` function is your friend here :)
+
+```js
+dbg.helpers.findSource("todo-view")
+/*
+{
+  "isPrettyPrinted": false,
+  "loadedState": "loaded",
+  "text": "/*global B",
+  "sourceMapURL": null,
+  "isWasm": false,
+  "url": "http://firefox-dev.tools/debugger-examples/examples/todomvc/js/views/todo-view.js",
+  "contentType": "text/javascript",
+  "isBlackBoxed": false,
+  "id": "server1.conn16.child1/source33"
+}
+*/
+```
+
+#### sendPacket
+
+`dbg.sendPacket` is an easy way to send a packet to the server. This is a useful helper for prototyping new APIs that aren't used in the UI yet.
```

Comment:
this looks like it should be `dbg.helpers.sendPacket`

also, "an easy way" suggests there is another way... maybe to avoid confusion this can say: 

"dbg.sendPacket communicates with the server using the packet pattern outlined [here under "packets"](http://docs.firefox-dev.tools/backend/protocol.html)..."

in case we want people to have more context. plus linking is great!